Learn/js/string_script.js

```diff
@@ -0,0 +1,29 @@
+function rotateString(){
+ var start = document.getElementById("rotate"). value;
+ var timer = setInterval(
+ function () 
+ {
+  start = start[start.length-1] + start.substring(0, start.length - 1);
+  document. getElementById( "resRotate" ).innerHTML = start;
+ }, 500);
+ setTimeout(function() {
+  clearInterval(timer);
+  alert('enough');
+  }, 5000);
+}
+function Year(){
```

Comment:
try to use camel case for function and calass names. Here more datails https://www.w3schools.com/js/js_conventions.asp

Reply:
renamed to fillLeapYearElement